File: Add Code Here/JAVA/Matrix Addition.java
import java.util.Scanner;
 
public class JavaMatrixAddition {
public static void main(String args[]){
int i,j,m,n;
Scanner sc = new Scanner(System.in);
System.out.println("Enter number of rows and columns:");
m = sc.nextInt();
n = sc.nextInt();
int a[][] = new int[m][n];
int b[][] = new int[m][n];
int c[][] = new int[m][n];
//reading first matrix
System.out.println("\nEnter elements of first matrix row wsie:");
for(i = 0; i < m; ++i){
for(j = 0; j < n; ++j){
a[i][j] = sc.nextInt();
}
}
//reading second matrix
System.out.println("\nEnter elements of second matrix row wsie:");
for(i = 0; i < m; ++i){
for(j = 0; j < n; ++j){
b[i][j] = sc.nextInt();
}
}
 
//performing addition of two matrices
for(i = 0; i < m; ++i){
for(j = 0; j < n; ++j){
c[i][j] = a[i][j] + b[i][j];
}
}
//printing the resultant matrix
System.out.println("\nAddtion of two matrices is:");
for(i = 0; i < m; ++i){
for(j = 0; j < n; ++j){
System.out.print(c[i][j] + " ");
}
System.out.print("\n");
}
}
}

File: Add Code Here/PYTHON/matrix addition.py
print("Enter Row and Column Size of First Matrix: ", end="")
rowOne = int(input())
colOne = int(input())
print("Enter Row and Column Size of Second Matrix: ", end="")
rowTwo = int(input())
colTwo = int(input())

if rowOne==rowTwo and colOne==colTwo:
    matOne = []
    print("\nEnter " +str(rowOne*colOne)+ " Elements for First Matrix: ")
    for i in range(rowOne):
        matOne.append([])
        for j in range(colOne):
            num = int(input())
            matOne[i].append(num)

    matTwo = []
    print("\nEnter " +str(rowTwo*colTwo)+ " Elements for Second Matrix: ")
    for i in range(rowTwo):
        matTwo.append([])
        for j in range(colTwo):
            num = int(input())
            matTwo[i].append(num)

    matThree = []
    for i in range(rowOne):
        matThree.append([])
        for j in range(colTwo):
            matThree[i].append(matOne[i][j]+matTwo[i][j])

    print("\nAddition Result:")
    for i in range(rowOne):
        for j in range(colOne):
            print(matThree[i][j], end=" ")
        print()
else:
    print("\nDimension Mismatched!")
